packages/tools/pactjs-cli/src/utils/callLocal.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from 'cross-fetch';
+
+export async function callLocal(
+  apiHost: string,
+  body: string,
+): Promise<{
```

Comment:
Not sure but can't you do something like `ReturnValueOf<fetch>`?

Reply:
Don't think so. We're not returning the response of fetch as-is, but we're building our own object.